matrimony_admin/views.py

```
@@ -1,16 +1,22 @@
-from django.views.generic import TemplateView, DetailView, ListView
+from django.shortcuts import render, redirect, get_object_or_404, get_object_or_404
+from django.views.generic import TemplateView, DetailView, ListView,DeleteView,UpdateView
 from django.contrib import messages
 from django.contrib.auth import authenticate, login, logout
-from django.views.generic import FormView
+from django.views.generic import FormView, ListView
 from django.urls import reverse_lazy
-from .forms import AdminLoginForm, AdminProfileForm, NotificationDetailsForm
+from .forms import AdminLoginForm, AdminProfileForm,NotificationDetailsForm, EditUserForm
 from .models import BlockedUserInfo
 from U_auth.permissions import *
```

Ruff

`from U_auth.permissions import *` used; unable to detect undefined names

(F403)